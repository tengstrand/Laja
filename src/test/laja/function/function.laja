
#import "src/test/laja/function/Incrementer.groovy"

#set (incrementer = new Incrementer())

#function increment(x)
  #return (incrementer.increment(x))
#end

#write "target/test-output/function.out"
  #set (a = increment(1))
  #increment(10) into value
a = {a}
value = {value}
#end
