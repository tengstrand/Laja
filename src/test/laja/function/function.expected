a = 2
value = 11
